### ISSUE TITLE
Apache IP Binding 127.0.0.1 no longer working. Resulting in ERROR: Connection disallowed

### ISSUE THREAD

**Issue by Frequency-Overload** [nextcloud/all-in-one#4667]
### Steps to reproduce
**1.** run nextcloud compose file
[nextcloud compose file.txt](https://github.com/nextcloud/all-in-one/files/15360184/nextcloud.compose.file.txt)

```
services:
  nextcloud-aio-mastercontainer:
    image: nextcloud/all-in-one:latest
    init: true
    restart: always
    container_name: nextcloud-aio-mastercontainer
    volumes:
      - nextcloud_aio_mastercontainer:/mnt/docker-aio-config
      - /var/run/docker.sock:/var/run/docker.sock:ro
    ports:
      - "127.0.0.1:8080:8080"
    environment:
      APACHE_PORT: 11000
      APACHE_IP_BINDING: 127.0.0.1
      NEXTCLOUD_UPLOAD_LIMIT: 20G
volumes:
  nextcloud_aio_mastercontainer:
    name: nextcloud_aio_mastercontainer
```

**2.** run caddy compose file
[caddy compose file.txt](https://github.com/nextcloud/all-in-one/files/15360188/caddy.compose.file.txt)

```
services:
  caddy:
    image: caddy:latest
    restart: always
    network_mode: "host"
    volumes:
      - ./Caddyfile:/etc/caddy/Caddyfile
      - ./site:/srv
      - ./caddy_data:/data
      - ./caddy_config:/config
volumes:
  caddy_data:
    external: true
  caddy_config:
    external: true
```

**3.** Caddyfile
[Caddyfile.txt](https://github.com/nextcloud/all-in-one/files/15360191/Caddyfile.txt)

```
https://nextcloud.example.com:443 {
    reverse_proxy localhost:11000
}
```

**4.** Navigate to ip adress:8080. Paste generated password on the “Open Nextcloud AIO login” page
**5.** Submit your domain on the “New AIO instance” page 
**6.** Choose any optional containers, and then save the changes. Submit a time zone.
**7.** Click “Download and start containers” 
**8.** Download is completed. Page is refreshed. Click “Open your Nextcloud” 

### Expected behavior 
Several months back I was successful at installing Nextcloud with my preferred settings of:
APACHE_PORT: 11000
APACHE_IP_BINDING: 127.0.0.1 
Then on the final screen of the install…. Click “Open your Nextcloud”, and be presented with the login page.

### Actual behavior 
Click “Open your Nextcloud” link on the final page of the install. 
The result is:

**Service Unavailable**
"The server is temporarily unable to service your request due to maintenance downtime or capacity problems. Please try again later."

Last few lines in the log file for nextcloud-aio-nextcloud:
listen.allowed_clients = 127.0.0.1,::1,172.26.0.7,172.26.0.2
+ set +x
[17-May-2024 23:19:54] NOTICE: fpm is running, pid 694
[17-May-2024 23:19:54] NOTICE: ready to handle connections
ERROR: Connection disallowed: IP address '172.26.0.1' has been dropped.

### Debian_Version: 12.5 
kernel version : 6.1.0-21-amd64
Codename: bookworm

#### Other valuable info 

**Debugging:**
From within the proxy container run nc -z localhost 11000; echo $?
output
= 0

**LOGS**

nextcloud-aio-nextcloud log with APACHE_IP_BINDING 127.0.0.1 
[APACHE_IP_BINDING 127.0.0.1 LOG.txt](https://github.com/nextcloud/all-in-one/files/15360302/APACHE_IP_BINDING.127.0.0.1.LOG.txt)

nextcloud-aio-nextcloud  log with APACHE_IP_BINDING 0.0.0.0 
[APACHE_IP_BINDING 0.0.0.0 LOG.txt](https://github.com/nextcloud/all-in-one/files/15360305/APACHE_IP_BINDING.0.0.0.0.LOG.txt)

nextcloud-aio-apache LOG
[nextcloud-aio-apache log.txt](https://github.com/nextcloud/all-in-one/files/15360399/nextcloud-aio-apache.log.txt)

Using the following in the compose file, results in a successful install
APACHE_PORT: 11000
APACHE_IP_BINDING: 0.0.0.0

**Docker - Engine & Compose version:**
[Docker - Engine & Compose version.txt](https://github.com/nextcloud/all-in-one/files/15360312/Docker.-.Engine.Compose.version.txt)

**Docker network inspect**

nextcloud_default
[nextcloud_default.txt](https://github.com/nextcloud/all-in-one/files/15360322/nextcloud_default.txt)

nextcloud-aio
[nextcloud-aio.txt](https://github.com/nextcloud/all-in-one/files/15360326/nextcloud-aio.txt)

**Comment by szaimen** [comment]
Hi, thanks for the report! Can you do the following if you should not need ipv6 support and check if that fixes the problem (while keeping `APACHE_IP_BINDING: 127.0.0.1`?

Add the following config to /etc/docker/daemon.json and then restart docker:
```json
{
    "userland-proxy": false
}
```

**Comment by Frequency-Overload** [comment]
Thank you for your reply,
I added the following config to /etc/docker/daemon.json and then restart docker:

{
    "userland-proxy": false
}

Was met with the same error “ERROR: Connection disallowed: IP address '172.22.0.1' has been dropped.”
Here are the log files for both nextcloud-aio-nextcloud and nextcloud-aio-apache

[nextcloud-aio-apache log.txt](https://github.com/nextcloud/all-in-one/files/15399712/nextcloud-aio-apache.log.txt)
[nextcloud-aio-nextcloud log.txt](https://github.com/nextcloud/all-in-one/files/15399714/nextcloud-aio-nextcloud.log.txt)

**Comment by szaimen** [comment]
This is now released with v8.3.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
